gc/base/MemoryPoolSplitAddressOrderedListBase.cpp

```diff
@@ -54,25 +54,19 @@
 #include "SweepPoolState.hpp"
 
 bool
-J9ModronHeapFreeList::initialize(MM_EnvironmentBase* env)
+J9ModronFreeList::initialize(MM_EnvironmentBase* env)
```

Comment:
Was the class name wrong?

Reply:
FWIW, it would not be hard to change the name to `J9ModronHeapFreeList`.  The usages of this class are in just a few places:
```c++
gc/base/MemoryPoolHybrid.cpp:12
gc/base/MemoryPoolHybrid.hpp:7
gc/base/MemoryPoolSplitAddressOrderedListBase.cpp:7
gc/base/MemoryPoolSplitAddressOrderedListBase.hpp:4
````
Any preference to a name?

Reply:
keep your current changes as I had missed the mismatch in naming in the original code.

Reply:
I would leave it the way you have it now.  I had originally missed the mismatch in names.  This is ok.